build.gradle

```diff
@@ -151,7 +151,7 @@ subprojects {
         ignoreFailures = Boolean.getBoolean("ignoreTestFailures")
         include "**/*Tests.*"
         exclude "**/*Abstract*.*"
-        classpath = it.classpath + files("${projectDir}/src/main/resources") + files("${projectDir}/src/test/resources")
+        classpath = it.classpath + files("${projectDir}/src/main/resources") + files("${projectDir}/src/test/resources") + files("${projectDir}/../../webapp/resources")
```

Comment:
This shouldn't be needed if I move the static-uris.properties file into a jar, presumably the same jar where it is used. I will try to update the pull request sometime in the next few days.